src/test/java/com/epam/tc/hw2/exercise2/SiteSelectorsTests.java

```diff
@@ -0,0 +1,288 @@
+                          webDriver.findElement(By.xpath(xpathFirstLogElement)).getText())
+                      .as("Correct log message when selected Selen selector")
+                      .contains("metal: value changed to Selen");
+
+        // test dropdown
+
+        // change current selection to prevent intersection with first check
+        dropdownSelectorColors.selectByVisibleText("Yellow");
+
+        dropdownSelectorColors.selectByVisibleText("Red");
+        selectedColor = dropdownSelectorColors.getFirstSelectedOption();
+        softAssertions.assertThat(selectedColor.getText())
+                      .as("Selected color is Red")
+                      .isEqualTo("Red");
+        softAssertions.assertThat(
+                          webDriver.findElement(By.xpath(xpathFirstLogElement)).getText())
+                      .as("Correct log message when selected Red in dropdown menu 'colors'")
+                      .contains("Colors: value changed to Red");
+
+        dropdownSelectorColors.selectByVisibleText("Green");
+        selectedColor = dropdownSelectorColors.getFirstSelectedOption();
+        softAssertions.assertThat(selectedColor.getText())
+                      .as("Selected color is Green")
+                      .isEqualTo("Green");
+        softAssertions.assertThat(
+                          webDriver.findElement(By.xpath(xpathFirstLogElement)).getText())
+                      .as("Correct log message when selected Green in dropdown menu 'colors'")
+                      .contains("Colors: value changed to Green");
+
+        dropdownSelectorColors.selectByVisibleText("Blue");
+        selectedColor = dropdownSelectorColors.getFirstSelectedOption();
+        softAssertions.assertThat(selectedColor.getText())
+                      .as("Selected color is Blue")
+                      .isEqualTo("Blue");
+        softAssertions.assertThat(
+                          webDriver.findElement(By.xpath(xpathFirstLogElement)).getText())
+                      .as("Correct log message when selected Blue in dropdown menu 'colors'")
+                      .contains("Colors: value changed to Blue");
+
+        dropdownSelectorColors.selectByVisibleText("Yellow");
+        selectedColor = dropdownSelectorColors.getFirstSelectedOption();
+        softAssertions.assertThat(selectedColor.getText())
+                      .as("Selected color is Yellow")
+                      .isEqualTo("Yellow");
+        softAssertions.assertThat(
+                          webDriver.findElement(By.xpath(xpathFirstLogElement)).getText())
+                      .as("Correct log message when selected Yellow in dropdown menu 'colors'")
+                      .contains("Colors: value changed to Yellow");
+
+        // 10. Close Browser
+        webDriver.close();
+        assertThatThrownBy(() -> webDriver.getTitle()).as("Browser is closed")
+                                                      .isInstanceOf(NoSuchSessionException.class);
```

Comment:
Browser should be closed in the `@AfterMethod`

Reply:
If we have only 1 page opened and close it, the browser will close automaticaly.
And I thought that all steps that described in Execise2 should be in methods with @Test annotations.
(step 12 in Execise1, step 10 in Execise2)

Reply:
fixed